pillar/edx/xqwatcher.sls

```diff
@@ -98,6 +98,44 @@ edx:
       {% endfor %}
       {% endif %}
       {% endfor %}
+      {% set 686_xqueue_creds = salt.vault.read('secret-residential/global/course-686-xqueue-credentials') %}
+      - COURSE: mit-686x
+        GIT_REPO: git@github.mit.edu:mitx/graders-mit-686x
+        GIT_REF: master
+        PYTHON_REQUIREMENTS:
+          - name: numpy
+            version: 1.14.0
+          - name: numpydoc
+            version: 0.7.0
+          - name: pandas
+            version: 0.22.0
+          - name: pandocfilters
+            version: 1.4.2
+          - name: scikit-image
+            version: 0.13.1
+          - name: scikit-learn
+            version: 0.19.1
+          - name: scipy
+            version: 1.0.0
+          - name: matplotlib
+            version: 2.1.2
+        PYTHON_EXECUTABLE: /usr/bin/python3
+        QUEUE_NAME: mitx-686xgrader
+        QUEUE_CONFIG:
+          SERVER: https://xqueue.edx.org
+          CONNECTIONS: 5
+          HANDLERS:
+            - HANDLER: 'xqueue_watcher.jailedgrader.JailedGrader'
+              CODEJAIL:
+                name: mit-686x
+                user: mit-686x
+                lang: python3
+                bin_path: '{% raw %}{{ xqwatcher_venv_base }}{% endraw %}/mit-686x/bin/python'
+              KWARGS:
+                grader_root: ../data/mit-686x/graders/
```

Comment:
Should this path be consistent with previous `../data/mit-686x-{{ purpose }}-{{ queue_name }}/graders/python3graders/`?

Reply:
No, because there will only be one instance of this grader.